README.md

```diff
@@ -1,2 +1,272 @@
 # TIDES
-Transit ITS Data Exchange Specification for historical ITS data
+Transit ITS Data Exchange Specification (TIDES) for historical ITS data.
+
+This repository references TIDES as described in the [prepublication report](https://nap.nationalacademies.org/catalog/26674/improving-access-and-management-of-public-transit-its-data).
+
+
+## Files in Data Structure
```

Comment:
Can we translate this to a data spec standard (i.e. json-schema or frictionless data package which is better for tabular data ) and then auto-generate the markdown from that (there are packages [but also not hard on your own...](https://github.com/ianberg-volpe/GMNSpy/blob/7d750063c2db06085fedc82d24af5a9c4ae9945f/gmnspy/schema.py#L101)

Reply:
Goal would be for computer-readable stuff to exist in 'code' and  human-facing things to be generated in a site (i.e. https://e-lo.github.io/GMNSpy/spec.html)

Reply:
I agree that I would like to see the tables each broken out into separate files as encoded schemas of some sort.

Reply:
> Goal would be for computer-readable stuff to exist in 'code' and human-facing things to be generated in a site (i.e. https://e-lo.github.io/GMNSpy/spec.html)

Got it, thanks for the guidance.